assets/stylesheets/style.scss

```diff
@@ -6,7 +6,7 @@
 //@import 'shared/colors'; // import all of our wpcom colors
 @import 'components/dialog/style';
 
-#wc-connect-admin-container,
+.wc-connect-container,
```

Comment:
Could we keep the `admin` part please? When we add front-side containers (e.g. in my-account or elsewhere) it will probably be useful to be able to distinguish Calypso like styles from theme inspired styles.